java/core/src/java/org/apache/orc/impl/ConvertTreeReaderFactory.java

```diff
@@ -1068,6 +1068,8 @@ public void nextVector(ColumnVector previousVector,
   }
 
   public static class StringGroupFromBooleanTreeReader extends StringGroupFromAnyIntegerTreeReader {
+    private static final byte[] TRUE_BYTES = "TRUE".getBytes(StandardCharsets.US_ASCII);
+    private static final byte[] FALSE_BYTES = "FALSE".getBytes(StandardCharsets.US_ASCII);
```

Comment:
In my opinion, it would be better to continue to use the UTF_8 charset to reduce the impact of this change.


Is there a benefit to not using it? My understanding is that the final result should be more or less equivalent because the letters are part of ASCII but I'm not an expert on UTF8 codepoints.

Reply:
The two will produce the same output since all numeric values (0-9) are ASCII values.  There is slightly less overhead with using ASCII directly as opposed to using UTF8 which will discover that all values are ASCII.